Task1/magic.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>magic</title>
+
+    <style>
+        a {
+            text-decoration: underline;
+            /* Подчеркивание заголовка */
+        }
+        
+        a {
+            font-style: italic;
+        }
+    </style>
+</head>
+
+<body>
+    <p>
+        <h1>Волшебные формулы</h1></p>
+    <p>(x+y)<sup>3</sup>=x<sup>3</sup>+3x<sup>2</sup>y+3xy<sup>2</sup>+y<sup>3</sup> </p>
+    <p>∆x <sub>1</sub>+∆x<sub>2</sub>+..+∆x<sub>n</sub>=∑∆x<sub>k</sub> </p>
+    <p>Если ∀ x ∈ R такого, что |x|≤1 ∃y ≥1, то č̣̓
+        <∞ </p>
+            <hr>
+            <a href="http://shpargalkablog.ru/2014/03/mathematical-formula-html.html">© Сайт с формулами </a>
```

Comment:
use html-code for symbols https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references